libs/coroutines/include/hpx/coroutines/detail/context_base.hpp

```diff
@@ -204,6 +214,11 @@ namespace hpx { namespace threads { namespace coroutines { namespace detail {
             delete_tss_storage(m_thread_data);
 #else
             m_thread_data = 0;
+#endif
+#ifdef HPX_HAVE_LIBCDS
```

Comment:
Minor style detail: We tend to use `#if defined(X)` instead of `#ifdef X`.

Reply:
Thanks, I just adapted the style. Please check again.